### PR TITLE
Use fixed php version

### PR DESCRIPTION
**Pull request by greg0ire**
We don't want to use phpunit 6 yet.

### CODE REVIEW

File: project/.travis/install_test.sh [hunk 1]
```diff
@@ -5,9 +5,9 @@ mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
 if [ ${TRAVIS_PHP_VERSION} '<' '5.6' ]; then
-    PHPUNIT_PHAR=phpunit-old.phar
+    PHPUNIT_PHAR=phpuniti-4.8.9.phar
```
Comment (core23) [inline project/.travis/install_test.sh]: Do we need the patch level? `phpuniti-4.8.phar` should work too

File: project/.travis/install_test.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@ mkdir --parents "${HOME}/bin"
...
 else
-    PHPUNIT_PHAR=phpunit.phar
+    PHPUNIT_PHAR=phpunit-5.7.9.phar
```
Comment (core23) [inline project/.travis/install_test.sh]: Same here

File: project/.travis/install_test.sh (hunk 1 above)
Comment (greg0ire) [inline project/.travis/install_test.sh]: It's not listed, but indeed, there's a nice 302.

**Comment by core23** [comment]
What about adding phpunit as a dependency https://github.com/sonata-project/dev-kit/issues/197 ...?

**Comment by greg0ire** [comment]
You mean using simple-phpunit? I thought about it, but right now I can't (too long), so here is a quick fix before I look into it.

File: project/.travis/install_test.sh (hunk 1 above)
Comment (greg0ire) [inline project/.travis/install_test.sh]: fixed

File: project/.travis/install_test.sh [hunk 3]
```diff
@@ -5,9 +5,9 @@ mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
 if [ ${TRAVIS_PHP_VERSION} '<' '5.6' ]; then
-    PHPUNIT_PHAR=phpunit-old.phar
+    PHPUNIT_PHAR=phpuniti-4.8.phar
```
Comment (Soullivaneuh) [inline project/.travis/install_test.sh]: Typo here: phpunit~i~

Comment (greg0ire) [inline project/.travis/install_test.sh]: Good catch!

**Comment by core23** [comment]
The commit message was wrong. It's not about `php` versions, but `phpunit` versions.

**Comment by greg0ire** [comment]
I'm very tired, sorry…

**Comment by greg0ire** [comment]
I hope there are no other fuck ups…

**Comment by core23** [comment]
Didn't see that either...

**Comment by greg0ire** [comment]
I'm trying out simple-phpunit now if anyone's interested https://github.com/sonata-project/SonataAdminBundle/pull/4307

**Pull request merged**